src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -227,11 +243,8 @@ public function writeCssFileForPortal($css, $portal)
     {
         $lessDir = $this->getLocalPathToCompiledLess();
 
-        if (false === \is_dir($lessDir)) {
-            if (!mkdir($lessDir, 0777, true) && !\is_dir($lessDir)) {
-                return false;
-            }
-        }
+        // NOTE this is probably already done in getGeneratedCssForPortal() for the cache of Less itself
```

Comment:
I think this comment is a bit cryptic. What we should say is (correct me if I'm wrong) that this call ensures the directory is created even if a) `getGeneratedCssForPortal` wasn't called before (which reduces temporal coupling) or b) the cache directory is no subdir of the output dir for any reasons.

Reply:
removed